ush/set_ozone_param.py

```diff
@@ -90,8 +95,10 @@ def set_ozone_param(ccpp_phys_suite_fp):
         fixgsm_ozone_fn = "global_o3prdlos.f77"
         ozone_param = "ozphys"
     else:
-        raise KeyError(f'Unknown or no ozone parameterization specified in the '
-                        'CCPP physics suite file "{ccpp_phys_suite_fp}"')
+        raise KeyError(
+            f"Unknown or no ozone parameterization specified in the "
+            'CCPP physics suite file "{ccpp_phys_suite_fp}"'
```

Comment:
For consistency, would it be best to convert the single quotes to double quotes, then convert the double quotes to single quotes around `{ccpp_phys_suite_fp}`?